ui/app/macros/maybe-query-record.js

```diff
@@ -7,6 +7,23 @@ import { computed } from '@ember/object';
 import ObjectProxy from '@ember/object/proxy';
 import PromiseProxyMixin from '@ember/object/promise-proxy-mixin';
 import { resolve } from 'rsvp';
+/**
+ * after upgrading to Ember 4.12 a secrets test was erroring with "Cannot create a new tag for `<model::capabilities:undefined>` after it has been destroyed"
```

Comment:
could be, but I thought in the app we saw an id instead of `undefined` 🤔 FWIW tests were failing with `A record in a disconnected state cannot utilize the store. This typically means the record has been destroyed, most commonly by unloading it.` when I ran the test suite reverting this change but keeping all others. 
The good/bad news is that we aren't the only ones seeing this. But, it seems to be most problematic in tests but making a note here to test the application with varying capabilities to make sure workflows aren't broken with this change

Reply:
Ah yeah, that error is familiar too. Sounds good!